elixir/mix.exs

```diff
@@ -1,16 +1,12 @@
 defmodule Firezone.MixProject do
   use Mix.Project
 
-  def version do
-    # Use dummy version for dev and test
-    System.get_env("APPLICATION_VERSION", "0.0.0+git.0.deadbeef")
-  end
-
   def project do
     [
       name: :firezone,
       apps_path: "apps",
-      version: version(),
+      # mark:automatic-version
+      version: "1.20231001.0",
```

Comment:
Maybe will be better to do `File.read!("VERSION")`? For the umbrella apps we can do the same: `File.read!("../../VERSION")`.

Reply:
Ah, I see your point, ok. I'll change it

Reply:
I'll save that change for the next PR if that's alright. This one's pretty large already